views/partials/_example.njk

```diff
@@ -6,12 +6,23 @@
   {% set examplePath = exampleRoot + "index.njk" %}
 {% endif %}
 {% set exampleURL = "/" + params.group + "/" + params.item + "/" + params.example + "/index.html" %}
-{% set exampleId = params.id | default("example-" + params.example) %}
+
 {% set exampleTitle = getFrontmatter(examplePath).title %}
+{% if params.titleSuffix %}
+  {% set exampleTitle = exampleTitle + " " + params.titleSuffix %}
+{% endif %}
+{% set exampleTitle = exampleTitle + " example" %}
+
+{% if params.id %}
```

Comment:
I don't think there are any examples that pass through 'id' now so this may be unnecessary.

Reply:
`id` is still used by at least header and footer examples.

Reply:
I wonder if it's necessary though given that we can generate a unique id from the unique title?

Not blocking just thought this could simplify the example partial a bit